src/main/lrsql/system/webserver.clj

```diff
@@ -64,7 +64,7 @@
          config
          (:config lrs))
         ;; LRS reaction toggle
-        {:keys [enable-reactions]}
```

Comment:
This is no longer _just_ about LRS reactions toggling, isn't it ;)

Reply:
fixed, but i would really focus in the UI one ;)